wp-content/themes/haruoshi_origin/header.php

```diff
@@ -12,3 +12,4 @@
 <header class="site-header">
     header
 </header>
+<main>
```

Comment:
classはスタイルを当てるため、roleはアクセシビリティのために指定します
```suggestion
<main class="main" role="main">
```

Reply:
ほほう、mainってbodyみたいな感じかと思ってた

Reply:
ほぼそうだと思ってくれていいよ